### PR TITLE
passing selected row to the prediction result table

### LINKED ISSUE #3090: Keras model prediction supports saving the column data which is not inside the features
The following SQL programs would raise an error when predicting. It is because we do not support saving the column data which is not inside the features (i.e., `petal_width` in the following example).

```sql
SELECT sepal_length, sepal_width, petal_length
FROM iris.train
TO TRAIN sqlflow_models.OneClassSVM
WITH model.kernel = "rbf"
INTO iris.my_model;

SELECT sepal_length, sepal_width, petal_length, petal_width
FROM iris.train
TO PREDICT iris.predict_result.label
USING iris.my_model;
```

### PR DESCRIPTION
**Pull request by Yancey0623**
fixed #3090

### CODE REVIEW

File: python/runtime/tensorflow/predict.py [hunk 1]
```diff
@@ -108,8 +108,11 @@ def eval_input_fn(batch_size, cache=False):
     column_names.append(result_col_name)
 
     column_names.extend(extra_result_cols)
+
     with db.buffered_db_writer(conn, result_table, column_names, 100) as w:
-        for features in pred_dataset:
+        for features, (row, _) in zip(pred_dataset, predict_generator()):
```
Comment (sneaxiy) [inline python/runtime/tensorflow/predict.py]: I wonder if the result order of the `SELECT ...` statement is always deterministic? I mean: does the data order of `pred_dataset` can always be the same as `predict_generator`?

Comment (Yancey0623) [reply python/runtime/tensorflow/predict.py]: `dataset` is generated by `generator`, according to  the source code of [from_generator](https://github.com/tensorflow/tensorflow/blob/v2.3.1/tensorflow/python/data/ops/dataset_ops.py#L823), it stays the same order expect calling [shuffle](https://www.tensorflow.org/api_docs/python/tf/data/Dataset#shuffle)

**Review by sneaxiy — APPROVED** [review]
LGTM.
